src/components/ViewDao/DaoRedemptionsContainer.tsx

```diff
@@ -36,34 +36,42 @@ class DaoRedemptionsContainer extends React.Component<IProps, null> {
     // TODO: the reward object from the subgraph only gives rewards for voting and staking and dao bounty,
     // the original code also considers ethREward and externalTokenRewards
     // let ethReward = 0
+
+    // TODO: move all the next logic to the client library
+    // calculate the total rewards from the genesisprotocol
     const genReward = new BN("0");
+    const ethReward = new BN("0");
     const reputationReward = new BN(0);
     // , externalTokenReward = 0;
```

Comment:
shouldnt we add in external token rewards now too?

Reply:
perhaps, I don't know, there could be 100 different tokens, so it's a whole new ballgame.